db/migrate/20220607150432_disallow_null_emoji_on_user_status.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class DisallowNullEmojiOnUserStatus < ActiveRecord::Migration[7.0]
+  def up
+    execute "UPDATE user_statuses SET emoji = 'mega'"
```

Comment:
📣 the megaphone feels like a very loud choice for a default emoji. Maybe 💬 (speech_balloon) is a better default? Or 🗨️ (left_speech_bubble).